meetings/2022-03-09.md

```diff
@@ -0,0 +1,134 @@
+* Elf: In that case (browser) the Solid storage/RS would act as a OAuth client. You'd still have a client.
+* Matthieu: How does that work with curl?
+* Elf: Then curl is your client.
+* Matthieu: How do you uniquely identify those?
+* Elf: You could use client credentials and register the client with your OP and curl would act as that client.
+* Matthieu: And typically client credentials flow doesn't issue uniquely identifiable client ID.
+* Elf: Yes, but even if it's not globally unique (we might still mandate it to be an IRI), even if not dereferenceable as a client ID. But the client would be user-bound. For Solid-OIDC, the client registration would bound the client credentials to a specific user. So the ID Token would have a specific WebID. We still need to define client credentials flow in Solid-OIDC.
+* Wouter: I pretty much agree that there always would be a client. The bigger question is is there always an agent or clients can act on their own behalf. I know that Pavlik believes there always will be agent/user. Which still doesn't answer delegation issue.
+* Pavlik: To clarify I think we always need agent/user, on whose behalf the application is acting. Mostly for any legal policies being applied to that user, software can't bare legal consequences.
+* Matthieu: I think it would be good to look at delegation issue independently from issue of syncing policies to ACL auxiliary resources.
+* Wouter: I think at one point we said that it will be hard to sync Data Grants with structure of ACL for example. At other point we said we would need UMA server to accept data grants as claims.
+* Elf: I'd see one of the ways to address delegation is to have push Data Grant as VC which an be verified as AC.
+* Wouter: I would rather first push the data grant VC to some server only responsible for access control resources based on data grants.
+* Elf: In a way the syncing server would still need a way to push claims.
+* Matthieu: From VC perspective, there can be any number of VC issuer. What I care about as an authorization server, which is aware of how to read ACL auxiliary resources, and interpret ACP. All I need to know that I will be granting access to anyone presenting VC of a specific type. Or maybe granting access to someone specifically presenting vc of certain type.
+* Elf: Where do you see client being presenting VC to?
+* Matthieu: Client would present it to AS, it would show VP to AS.
+* Elf: We currently leave the AS - RS coordination up to implementation.
+* Matthieu: No spec can rely of other spec implementing a translation layer for it to work. There need to be clear boundaries between specs.
+* Wouter: If Data Grans can be pushed to AS (as claims), but they are not meant to be translated to ACL auxiliary resources. That means that either Data Grant has final word or there even is no ACL resource behind it. Since they may conflict.
```

Comment:
```suggestion
* Wouter: If Data Grants can be pushed to AS (as claims), but they are not meant to be translated to ACL auxiliary resources, that means that either Data Grant has final word or there is no ACL resource behind it, since they may conflict.
```